src/Exceptionless.Core/Utility/MetricsConnectionString.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Exceptionless.Core.Utility {
```

Comment:
I think this belongs in the insulation project, the core project shouldn't know about any concrete implementations, only the abstraction.